pkg/pods/manifest_test.go

```diff
@@ -134,6 +134,21 @@ func TestRunAs(t *testing.T) {
 	Assert(t).AreEqual(manifest.RunAsUser(), "specialuser", "RunAsUser() didn't match expectations")
 }
 
+func TestLogExecutable(t *testing.T) {
+	config := testPod()
+	manifest, err := ManifestFromBytes(bytes.NewBufferString(config).Bytes())
```

Comment:
`string` and `[]byte` have [built-in conversions](https://golang.org/ref/spec#Conversions):

    ManifestFromBytes([]byte(config))